core/payment/src/api/debit_notes.rs

```diff
@@ -193,27 +199,45 @@ async fn send_debit_note(
     }
 
     let timeout = query.timeout.unwrap_or(params::DEFAULT_ACK_TIMEOUT);
-    with_timeout(timeout, async move {
+
+    let result = with_timeout(timeout, async move {
         match async move {
+            log::debug!(
+                "Sending DebitNote [{}] to [{}].",
+                debit_note_id,
+                debit_note.recipient_id
+            );
+
             ya_net::from(node_id)
                 .to(debit_note.recipient_id)
                 .service(PUBLIC_SERVICE)
                 .call(SendDebitNote(debit_note))
                 .await??;
             dao.mark_received(debit_note_id, node_id).await?;
+
```

Comment:
counter bumpin (line below) should be next to log msg you've added below. And pls also introduce counter for errors